AlphaWallet/Transactions/ViewControllers/TransactionViewController.swift

```diff
@@ -119,36 +120,38 @@ class TransactionViewController: UIViewController {
     private func configure() {
         buttonsBar.configure()
         let button = buttonsBar.buttons[0]
-        button.setTitle(R.string.localizable.moreDetails(), for: .normal)
+        button.setTitle(viewModel.detailsButtonText, for: .normal)
         button.addTarget(self, action: #selector(more), for: .touchUpInside)
 
         buttonsBar.isHidden = !viewModel.detailsAvailable
     }
 
-    private func item(title: String, value: String) -> UIView {
-        return  TransactionAppearance.item(
+    private func item(title: String, value: String, icon: UIImage? = nil) -> UIView {
+        return TransactionAppearance.item(
             title: title,
-            subTitle: value
-        ) { [weak self] in
-            self?.showAlertSheet(title: $0, value: $1, sourceView: $2)
+            subTitle: value,
+            icon: icon
+        ) { [weak self] _, _, _ in
+            self?.copy(value: value)
         }
     }
+    
+    @objc func copy(value: String) {
+        UIPasteboard.general.string = value
 
-    func showAlertSheet(title: String, value: String, sourceView: UIView) {
-        let alertController = UIAlertController(
-            title: nil,
-            message: value,
-            preferredStyle: .actionSheet
-        )
-        alertController.popoverPresentationController?.sourceView = sourceView
-        alertController.popoverPresentationController?.sourceRect = sourceView.bounds
-        let copyAction = UIAlertAction(title: R.string.localizable.copy(), style: .default) { _ in
-            UIPasteboard.general.string = value
-        }
-        let cancelAction = UIAlertAction(title: R.string.localizable.cancel(), style: .cancel) { _ in }
-        alertController.addAction(copyAction)
-        alertController.addAction(cancelAction)
-        present(alertController, animated: true, completion: nil)
+        let hud = MBProgressHUD.showAdded(to: view, animated: true)
+        hud.mode = .text
+        hud.label.text = viewModel.addressCopiedText
```

Comment:
- [ ] The bubble that appears shouldn't say "Address copied" when it's the "transaction #" or "Gas Fee", etc